test/unit/broadcast_active_set.c

```diff
@@ -104,24 +134,15 @@ int main(void)
 
     /* A total of npes tests are performed, where the root changes each time */
     for (i = 0; i < npes; i++) {
-        int j;
 
         if (me == i)
             printf(" + root %d\n", i);
 
-        shmem_broadcast64(dst, src, NELEM, i, 0, 0, npes, bcast_psync);
+        shmem_int64_broadcast(SHMEM_TEAM_WORLD, dst, src, NELEM, i);
```

Comment:
Could we do this here?
```suggestion
#ifdef ENABLE_DEPRECATED_TESTS
        shmem_broadcast64(dst, src, NELEM, i, 0, 0, npes, bcast_psync);
#else
        shmem_int64_broadcast(SHMEM_TEAM_WORLD, dst, src, NELEM, i);
#endif
```